libs/design/src/atoms/form/radio/radio.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { DaffRadioComponent } from './radio.component';
+import { Component, DebugElement } from '@angular/core';
+import { By } from '@angular/platform-browser';
+@Component({
+  template: `
+    <daff-radio [checked]=true value="testValue"></daff-radio>
+  `
+})
+class RadioWrapperComponent {}
+
+describe('DaffRadioComponent', () => {
+  let wrapper: RadioWrapperComponent;
+  let component: DaffRadioComponent;
+  let fixture: ComponentFixture<RadioWrapperComponent>;
+  let de: DebugElement;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        DaffRadioComponent,
+        RadioWrapperComponent
+      ]
+    })
+      .compileComponents();
+  }));
+
```

Comment:
Needs tests covering ControlValueAccessor implementation.

Reply:
Testing coverage was expanded in 7eff94fc15e2ee0f2adf31c5de274475d7e0c73d

Reply:
Let me know if additional tests could be added.